### ISSUE TITLE
[Feature Request] Changes to Add Indexer Window

### ISSUE THREAD

**Issue by kingfenix** [Prowlarr/Prowlarr#233]
**Is your feature request related to a problem? Please describe.**
I am always having to go back and forth from site to add indexer window to find out if its something I want to add or have added already.

**Describe the solution you'd like**
1. Hide indexers from add window if they are already added in main screen.
2. Provide in the add window the little "i" for information, that pops up and shows description same as on main screen to far right of indexer.

**Comment by kingfenix** [comment]
any updates on this request, I would think these two items are fairly easy to include/update and would make adding indexers SOOOOOOO much nicer.

**Comment by bakerboy448** [comment]
When there's an update to share an update will be posted

or more than likely this will either be closed as fixed or linked / referenced in a pull request then fixed once merged

**Comment by bakerboy448** [comment]
Addresses (sort of) Existing Indexers - https://github.com/Prowlarr/Prowlarr/commit/76daee3a1b864f461a90b7e494a2e7239976c140

**Comment by Qstick** [comment]
76daee3

![image](https://user-images.githubusercontent.com/376117/149856059-72ff451f-e652-40e0-b95b-9eb0eee92c7d.png)

**Comment by kingfenix** [comment]
yeah thats awesome, now if we could only get the clickable little "i" in there so I could view the sites link and other info to help decide if I want to add a indexer without closing and coming back

**Comment by bakerboy448** [comment]
I think adding the description to the list and perhaps the site as well would be beneficial and look cleaner than (i) popup?

not sure on mobile tho

**Comment by kingfenix** [comment]
yeah I can see maybe not wanting a modal inside another modal, just having the site link would make me happy cause I go in there every so often and see new sites and I am like well what do these have and I want to check the site to see if its something I want to add etc

________________________________

**Comment by Bullhogeley** [comment]
Maybe a sort on results where indexers that are not already added are at the top of the list?  Jackett does something similar but filters out indexers that you have already added and does not show them.

**Comment by Qstick** [comment]
Yea, we really don't want to completely filter them given Prowlarr supports adding an indexer more than once (say you wanted one for FL and one for normal, etc.). We can add a way to filter and un-filter them though.
